PWG/Cocktail/macros/AddMCEMCocktailV2.C

```diff
@@ -51,6 +52,7 @@ AliGenerator* AddMCEMCocktailV2(  Int_t collisionsSystem      = 200,
   gener->SelectMotherParticles(selectedMothers);
   gener->SetCollisionSystem(collisionsSystem);                //pp 7 TeV
   gener->SetCentrality(centrality);                           // kpp
+  gener->SetParametrizationFileV2Directory(paramV2FileDir);
   (AliPythia::Instance())->SetMSTU(22, pythiaErrorTolerance);   // tolerance for error due to rhos
```

Comment:
I would add here:
if (paramV2FileDir.CompareTo("")! = 0 ) gener->SetParametrizationFileV2Directory(paramV2FileDir);

instead of 
gener->SetParametrizationFileV2Directory(paramV2FileDir);